src/Components/StudentCard.jsx

```diff
@@ -0,0 +1,98 @@
+import React from "react";
+import { useState } from "react";
+import ShowMore from "./components/ShowMore";
```

Comment:
The C in `components` should be capitalized, since you named the folder `Components`